api/client/gro_client.py

```diff
@@ -301,38 +301,48 @@ def get_geo_centre(self, region_id):
 
             Example::
 
-                [{'centre': [ 45.7228, -112.996 ], 'regionId': 1215, 'regionName': 'United States'}]
-
+                [{'centre': [ 39.8333, -98.5855 ], 'regionId': 1215, 'regionName': 'United States'}]
         """
         return lib.get_geo_centre(self.access_token, self.api_host, region_id)
 
-    def get_geojsons(self, region_id, descendant_level=None):
+    def get_geojsons(self, region_id, descendant_level=None, zoom_level=7):
```

Comment:
just to confirm, we used to return simplified shapefile for all requests. is it intended to change the default to detailed boundaries table? @nemozen

Reply:
Yes, we want to keep the current server-side default, so we don't change
client side performance of the web app. for api clients, we want to default
to more detailed boundaries (as with shp files), and give  the client the
option to reduce the zoom level if higher level of accuracy is not needed.